From aea0d75c86bf8ef6043c9bf3bbcf69eced760331 Mon Sep 17 00:00:00 2001
From: Michael Reeves <michael.reeves077@gmail.com>
Date: Sun, 4 Oct 2026 21:12:23 +1100
Subject: [PATCH] main: update boot license message

Since m1n1 is GPL2+ licensed now, update log sent on boot.

Signed-off-by: Michael Reeves <michael.reeves077@gmail.com>
---
 src/main.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/main.c b/src/main.c
index dcac0b739..d16fc756e 100644
--- a/src/main.c
+++ b/src/main.c
@@ -143,7 +143,7 @@ void m1n1_main(void)
 {
     printf("\n\nm1n1 %s\n", m1n1_version);
     printf("Copyright The Asahi Linux Contributors\n");
-    printf("Licensed under the MIT license\n\n");
+    printf("Licensed under the GNU General Public License v2 or later\n\n");
 
     printf("Running in EL%lu\n\n", mrs(CurrentEL) >> 2);
 
